### ISSUE TITLE
Just downloaded; very confused.

### ISSUE THREAD

**Issue by Devix167** [rpgboss/rpgboss#481]
I have looked all over this program's features, but to no luck I have not found how to make a project without any of the built-in stuff.
I have also tried going into the database and trying to delete the built-in stuff from my project. I hit the delete key; nothing happened D:
I tried right clicking to make a menu pop up; no menu.
I did google searches; no luck.
I also did searches in the issues for this. No luck.
What do I do?

**Comment by EgyLynx** [comment]
Sorry... In build youre cant remove :(

**Comment by TerenasMenethil** [comment]
Yeah. The best you can do is download the tile sets, sprite sets, or face sets you wanted online, and just import them into your game. 

To import, just select "**Resources**". Then select a "**Resource Type**", and then click "_**Import**_".
